lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -179,13 +179,15 @@ class InputConfiguration {
     @required this.inputType,
     @required this.inputAction,
     @required this.obscureText,
+    @required this.autocorrect,
   });
 
   InputConfiguration.fromFlutter(Map<String, dynamic> flutterInputConfiguration)
       : inputType = EngineInputType.fromName(
             flutterInputConfiguration['inputType']['name']),
         inputAction = flutterInputConfiguration['inputAction'],
-        obscureText = flutterInputConfiguration['obscureText'];
+        obscureText = flutterInputConfiguration['obscureText'],
+        autocorrect = flutterInputConfiguration['autocorrect'];
```

Comment:
Is the framework guaranteed to send us a non-null value? If not, we may get an NPE in the `if` statement below.

Reply:
Framework always sends a non-null value.

Reply:
Link to text_input.dart: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/text_input.dart#L397